http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -116,12 +116,13 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       // Create namespace
       _ <- kubeAlg.createNamespace(kubeClient, kubernetesNamespace)
 
-      // If configured for the app type, call WSM to create a managed identity and postgres database.
-      // This returns a KSA authorized to access the database.
-      (maybeKsaFromDatabaseCreation, maybeDbName) <- maybeCreateWsmIdentityAndDatabase(app,
-                                                                                       params.workspaceId,
-                                                                                       params.landingZoneResources,
-                                                                                       kubernetesNamespace
+      // If configured for the app type, call WSM to create a managed identity and postgres database(s).
+      // This returns a KSA authorized to access the database(s).
+      (maybeKSAFromSharedDatabaseCreation, maybeSharedDbNames) <- maybeCreateWsmIdentityAndDatabases(
```

Comment:
This is where the identity and databases are created. Scroll down to see the body of this function